.spelling

```diff
@@ -460,6 +460,7 @@ xDS
 yaml
 yamls
 yournamespace
+zipkin
```

Comment:
Please add the term with the correct capitalization: Zipkin
If you use the `zipkin` command in the text, surround it with back-ticks \` like I just did.